### PR TITLE
Run e2e test as a flaky

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Run typescript e2e tests from pre-created image from quay.io/kkanova/downstream_e2e.

### CODE REVIEW

**Comment by Katka92** [comment]
[test]

File: dockerfiles/e2e-saas/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    #Download rh-che
+    cd ../ && \
+    git clone https://github.com/redhat-developer/rh-che.git && \
+    cd rh-che/e2e-saas && \
+    npm --silent i
+    
+WORKDIR /root/
+COPY docker-entrypoint.sh /root/
+
+ENTRYPOINT ["/root/docker-entrypoint.sh"]
```
Comment (ibuziuk) [inline dockerfiles/e2e-saas/Dockerfile]: no new line in the end

File: dockerfiles/e2e-saas/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+FROM quay.io/kkanova/upstream_e2e
```
Comment (ibuziuk) [inline dockerfiles/e2e-saas/Dockerfile]: what  is the base image about? and why do we need some custom personal e2e upstream image? @rhopp this looks like a hacky approach for me - what is the long-running plan/approach for e2e adoption in downstream?

File: dockerfiles/e2e-saas/docker-entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+fi
+
+# ------------------------------------------------------------------------------------------
+# -------------------- SET ALL NEEDED ENVIRONMENT VARIABLES --------------------------------
+# ------------------------------------------------------------------------------------------
+
+export TS_SELENIUM_USERNAME=$USERNAME
+export TS_SELENIUM_PASSWORD=$PASSWORD
+export TS_SELENIUM_BASE_URL=$URL
+#E2E_SAAS_TESTS_USER_TOKEN is already set
```
Comment (ibuziuk) [inline dockerfiles/e2e-saas/docker-entrypoint.sh]: do we need this comment? it does not clarify things for me personally

**Comment by ibuziuk** [comment]
We should not push this PR and depend on personal docker images. Basically, the first thing should be addressing all the review comments in https://github.com/eclipse/che/pull/13542 and got it merged in the upstream

File: dockerfiles/e2e-saas/Dockerfile (hunk 1 above)
Comment (Katka92) [reply dockerfiles/e2e-saas/Dockerfile]: Addressed.

File: dockerfiles/e2e-saas/Dockerfile (hunk 2 above)
Comment (Katka92) [reply dockerfiles/e2e-saas/Dockerfile]: Now the image is based on upstream docker image.

File: dockerfiles/e2e-saas/docker-entrypoint.sh (hunk 3 above)
Comment (Katka92) [reply dockerfiles/e2e-saas/docker-entrypoint.sh]: Comment removed.

File: dockerfiles/e2e-saas/google-chrome.repo [hunk 4]
```diff
@@ -0,0 +1,6 @@
+[google-chrome]
+name=google-chrome
+baseurl=http://dl.google.com/linux/chrome/rpm/stable/$basearch
+enabled=1
+gpgcheck=1
+gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub
```
Comment (ibuziuk) [inline dockerfiles/e2e-saas/google-chrome.repo]: @Katka92 no new line in the end

Comment (Katka92) [reply dockerfiles/e2e-saas/google-chrome.repo]: New line added.

**Pull request merged**